config/locales/fr.yml

```diff
@@ -405,6 +405,10 @@ fr:
       email_notifications_question: 'Comment puis-je modifier les alertes email?'
       email_notifications_answer: 'Rendez-vous sur la page <strong>%{accounts}</strong>. Vous pouvez modifier votre email, mot de passe, photo de profil, et bio ici. Vous pouvez aussi supprimer votre compte. Une fois votre compte supprimé, toutes vos données sont supprimées.'
     home:
+      update_pwd_modal:
+        title: 'Mettre à jour le mot de passe'
```

Comment:
```
Psych::SyntaxError: (<unknown>): did not find expected key while parsing a block mapping at line 409 column 9
```
@julianguyen I am getting above error while running `rake react_on_rails:locale`.
I think that Issue is in this line, not sure what I am missing.

Reply:
So the issue is actually with the next line:
```
'Nous avons mis en place de meilleures mesures de sécurité pour nos utilisateurs, s'il vous plaît %{update_password_link}'
```

The `'` in `s'l` is unescaped. You have to do:
```
'Nous avons mis en place de meilleures mesures de sécurité pour nos utilisateurs, s''il vous plaît %{update_password_link}'
```

Ahh sorry, this is hard the debug :S